packages/gluestack-ui/package.json

```diff
@@ -85,7 +85,14 @@
   },
   "peerDependencies": {
     "@gluestack-ui/themed": "^1",
-    "react": "*"
+    "react": "*",
+    "@formatjs/intl-getcanonicallocales": "*",
+    "@formatjs/intl-locale": "*",
+    "@formatjs/intl-pluralrules": "*",
+    "@formatjs/intl-numberformat": "*",
+    "@formatjs/intl-datetimeformat": "*",
+    "@formatjs/intl-listformat": "*",
+    "@formatjs/intl-relativetimeformat": "*"
```

Comment:
hmm, well I moved them away from the core lib because normally they would not be required under non-react native use cases: I really don't get what you mean with "without using gluestack-ui : this is in the gluestack-dedicated pacakge?

Reply:
I have used `@bonfhir/core` before in a React Native application, specifically because our GlueStack support at the time wasn't that great and my UI needs were not met by `@bonfhir/gluestack-ui`. That's when I realized how much formatters from `@bonfhir/core` depend on these `Intl` globals and how few of them are supported by the Hermes JS engine used by React Native.

I thought it might be helpful to declare the polyfills needed for React Native support for `@bonfhir/core` as optional peer dependencies because which polyfills are needed is an implementation details of those formatters. Since it's just peer dependencies, I'm not really willing to insist for this but I do think it's an appropriate choice.

I leave it up to it. I'm approving either way.

Reply:
aaah ok, I thought you wanted me to remove them from the gluestack stuff 😂 

will do that!